subprojects/internal-android-performance-testing/internal-android-performance-testing.gradle.kts

```diff
@@ -37,7 +37,7 @@ listOf(tasks.distZip, tasks.distTar).forEach {
     it { archiveBaseName.set("android-test-app") }
 }
 
-project(":distributions").tasks.register("buildDists") {
+tasks.register("buildDists") {
```

Comment:
The `buildTypes` seems to look for `buildDists` tasks everywhere. I don't know why it was attaching to the distribtuions project. I'm assuming it was legacy.